packages/server/src/fhir/search.test.ts

```diff
@@ -4213,7 +4215,7 @@ describe('FHIR Search', () => {
           resourceType: 'Composition',
           status: 'final',
           type: { text: 'test' },
-          date: '2025-01-01',
```

Comment:
`Composition.date` is a `dateTime` field, so it should have a date+time component.